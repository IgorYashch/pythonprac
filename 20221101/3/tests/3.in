f, g, h = Grange(32, 3, 8575), Grange(11, 5, 1042856), Grange(103, 11, 104577)
o, e = Grange(13, 22, 11), Grange(100, 10, 200)
print(f"{f=}, {g=}, {h=}")
print(f"{list(o)}({not(o)}), {list(e)}({not(e)})")
print(f"{len(f)=}, {len(g)=}, {len(h)=}")
print(f"{len(list(f))=}, {len(list(g))=}, {len(list(h))=}")
print(f"{f[2]=}, {h[33]=}")
a, b = f[3:3276], h[3:3267:5]
print(f"{a=}, {a[12]=}")
print(f"{b=}, {b[4]=}, {list(b)=}")