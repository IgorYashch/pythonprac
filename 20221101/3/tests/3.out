f=grange(32, 3, 8575), g=grange(11, 5, 1042856), h=grange(103, 11, 104577)
[](True), [100](False)
len(f)=6, len(g)=8, len(h)=3
len(list(f))=6, len(list(g))=8, len(list(h))=3
f[2]=288, h[33]=2392190905248444238523182083019840493
a=grange(3, 3, 3276), a[12]=1594323
b=grange(3, 161051, 3267), b[4]=2018249984797680027603, list(b)=[3]
